test/test_filters.rb

```diff
@@ -57,6 +57,20 @@ def select; end
       )
     end
 
+    should "markdownify with simple string in inline mode" do
+      assert_equal(
+        "one <em>simple</em> string",
+        @filter.markdownify("one _simple_ string", :inline)
+      )
+    end
+
+    should "markdownify with multi-line string in inline mode" do
+      assert_equal(
+        "first line\nsecond line",
+        @filter.markdownify("first line\n\nsecond line", :inline)
```

Comment:
What can you tell me about this test? Why two `\n` in the input and one `\n` in the output?